src/components/pager/pager-navigation.spec.js

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
-import Immutable from 'immutable';
 import 'jest-styled-components';
 import { ThemeProvider } from 'styled-components';
```

Comment:
You dont need `ThemeProvider` in unit tests if styled components have default theme applied

Reply:
removed.